tests/admin_views/models.py

```diff
@@ -625,7 +625,7 @@ class WorkHour(models.Model):
 
 class Question(models.Model):
     question = models.CharField(max_length=20)
-    posted = models.DateField(default=datetime.date.today)
+    posted = models.DateTimeField(default=datetime.date.today)
```

Comment:
It might be better to add a new field (something like `expires` could be fine). `default=datetime.date.today` isn't compatible with `DateTimeField`.